### PR TITLE
List default installed dependencies

### PR DESCRIPTION
**Pull request by astrojuanlu**
Taken from #7859, but removing mentions to deprecation policies or supported versions. Therefore, just describing what we do at the moment.

Comes from https://github.com/readthedocs/readthedocs.org/pull/8473#discussion_r708710557.

### CODE REVIEW

**Comment by astrojuanlu** [comment]
Rendered version: https://docs--8503.org.readthedocs.build/en/8503/default-versions.html

**Comment by stsewd** [comment]
I think we could move this to be a section of builds.rst

**Comment by astrojuanlu** [comment]
That was my first idea actually. If there are no objections I'll make the change 👍🏽

**Review by ericholscher — APPROVED** [review]
This is great. I agree `Builds` is probably the right place for it. That page in particular needs a lot of changes, so I'd love to have a refactor there as we go to add this, but we can keep them separate tasks, so this doesn't get blocked again.

**Comment by astrojuanlu** [comment]
Moved this content to the "Build process" page, and merging!

File: docs/tutorial/index.rst [hunk 1]
```diff
@@ -299,7 +299,7 @@ This has several advantages:
 - Some configurations are only available using the config file.
 
 Read the Docs works without this configuration
-by :doc:`making some decisions on your behalf </default-versions>`.
+by :ref:`making some decisions on your behalf <default-versions>`.
```
Comment (stsewd) [inline docs/tutorial/index.rst:302]: You can use `` :ref:`builds:Default versions of dependencies` `` instead of hardcoding the ref.

Comment (astrojuanlu) [reply docs/tutorial/index.rst:302]: Woops, I'm not used to have section autolabeling. thanks

**Pull request merged**